.github/workflows/build.yaml

```diff
@@ -46,7 +46,7 @@ jobs:
         # Forces libcurl to be statically linked by uninstalling the dynamic libs
         run: brew uninstall --ignore-dependencies curl
 
-      - run: ./gradlew ${{ matrix.task }}
+      - run: ./gradlew buildDependencies ${{ matrix.task }}
```

Comment:
To avoid that Gradle error
```suggestion
      - run: ./gradlew buildDependencies
      - run: ./gradlew ${{ matrix.task }}
```

Reply:
Is `mustRunAfter` not what I want? Seems like it won't make it so that `buildDependencies` always runs, but rather ensure ordering if they're both running.

Reply:
Yeah, that works too